mllib/src/test/scala/org/apache/spark/ml/feature/StringIndexerSuite.scala

```diff
@@ -178,6 +178,20 @@ class StringIndexerSuite
     }
   }
 
+  test("StringIndexer on column with empty string values") {
+    val data = sc.parallelize(Seq((0, "a"), (1, ""), (2, "c"), (3, "a"), (4, "a"), (5, "c")), 2)
+    val df = sqlContext.createDataFrame(data).toDF("id", "label")
```

Comment:
@jaceklaskowski Yes you are right. I'm using Intellij ide, which requires you add `import sqlContext.implicits._` in order to use syntax mentioned by you. Thus I was avoiding that.